src/tests/miscellaneous/mock-adb/common-adb-configs.js

```diff
@@ -163,13 +163,26 @@ function simulatePortForwardingError(oldConfig) {
     return cloneWithDisabledPattern(oldConfig, portForwardingCommandMatch);
 }
 
+const physicalDeviceName1 = 'device-1';
```

Comment:
Changes in this file are just to expose the device names for consumption by tests